templates/control/2fa/generate_recovery_codes.html

```diff
@@ -0,0 +1,51 @@
+$def with (tfa_recovery_codes, error)
+$:{RENDER("common/stage_title.html", ["Generate Recovery Codes", "Two-Factor Authentication", "/control/2fa/status"])}
+
+<div class="content form clear">
+  <form method="POST" class="form skinny clear" action="/control/2fa/generate_recovery_codes">
+    $if error == "2fa":
+      <div id="signin_error">
+        <strong>Whoops!</strong> The 2FA response token or recovery code you provided did not validate.<strong>
+        If entered, your recovery code was <strong>not</strong> consumed.
```

Comment:
``If entered, your recovery code was <strong>not</strong> consumed.``

Redundant line. If the code didn't validate, it wasn't correct, and therefore it wouldn't be consumed to begin with.